201-encrypt-running-windows-vm/azuredeploy.json

```diff
@@ -26,120 +14,137 @@
         "description": "Name of the KeyVault to place the volume encryption key"
       }
     },
-    "keyVaultResourceGroup": {
-      "type": "string",
-      "metadata": {
-        "description": "Resource group of the KeyVault"
-      }
-    },
-    "useExistingKek": {
-      "type": "string",
-      "defaultValue": "nokek",
-      "allowedValues": [
-        "nokek",
-        "kek"
-      ],
-      "metadata": {
-        "description": "Select kek if the secret should be encrypted with a key encryption key and pass explicit keyEncryptionKeyURL. For nokek, you can keep keyEncryptionKeyURL empty."
-      }
-    },
-    "keyEncryptionKeyURL": {
+    "keyVaultDomain": {
```

Comment:
remove this param and use the environment function on the var

Reply:
done